src/pages/extending-chat-widget/javascript-api/index.mdx

```diff
@@ -175,6 +178,31 @@ In order to hide particular greeting you can listen to [`new_event`](#on-new-eve
 LiveChatWidget.call("hide_greeting");
 ```
 
+## Trigger sales tracker
+
+It triggers a sales tracker created in LiveChat application and passes over data about a sale. The data is then visible in LiveChat's Reports and Archives. For more information about how Sales Tracker works and how to set it up on your website, check out our official Help Center article: https://www.livechat.com/help/sales-tracker/#how-sales-tracker-works-in-practice
```

Comment:
```suggestion
It triggers a sales tracker created in the LiveChat application and passes over data about a sale. The data is then visible in LiveChat's Reports and Archives. For more information about how Sales Tracker works and how to set it up on your website, check out our official Help Center article: https://www.livechat.com/help/sales-tracker/#how-sales-tracker-works-in-practice
```

Reply:
Also, about the lin -- could make it an external link that opens in a new tab?

Reply:
Done :)